contrib/automation_tests/orbit_capture_loading.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import LoadCapture
+from test_cases.main_window import VerifyModalWindowExistsAndClickOkay
+from test_cases.live_tab import AddIterator, VerifyFunctionCallCount
+from test_cases.capture_window import CheckTimers, CollapseTrack, ExpandTrack, FilterTracks, CheckThreadStates, \
+    ToggleCollapsedStateOfAllTracks
+
+"""Verify loading a capture in Orbit using pywinauto.
+
+There is no prerequisites before this script is run.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test it needs
+to by run from 64 bit python.
```

Comment:
```suggestion
to be run from 64 bit python.
```

Reply:
Or not, we have this typo in every single test I just realized.

Reply:
I fixed it here anyways.